powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/task/CleaningTask.java

```diff
@@ -46,23 +46,29 @@ public class CleaningTask {
     private final ActivationServiceBehavior activationServiceBehavior;
 
     @Scheduled(fixedRateString = "${powerauth.service.scheduled.job.uniqueValueCleanup:60000}")
-    @SchedulerLock(name = "expireUniqueValuesTask")
+    @SchedulerLock(
+            name = "expireUniqueValuesTask",
+            lockAtLeastFor = "#{T(java.lang.Math).round(${powerauth.service.scheduled.job.uniqueValueCleanup:60000} * 0.8)}")
```

Comment:
We cannot introduce a global value. My idea was to avoid adding other properties and set the value to 80 percent of the particular property.